### PR TITLE
Show review comment box only on a pull request page

### LINKED ISSUE #4625: Code review button doesn't work on commits.
- Gitea version (or commit ref): https://github.com/go-gitea/gitea/commit/7cb1c1cf20f1a07d98b78c2ba56c3b50777701a5
- Git version: 2.18.0
- Operating system: Linux
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [X] MySQL
  - [ ] MSSQL
  - [ ] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [X] Yes: https://try.gitea.io/wyattoday/Test1234/commit/31fb0641bc7f5d7f5421ec13790f8252c6439c1c
  - [ ] No
  - [ ] Not relevant
- Log gist:

## Description

New code-review feature doesn't work. This merged pull request: https://github.com/go-gitea/gitea/pull/3748

Click the "Review" button and nothing comes up. Neither does clicking the the lines to make code comments.

Presumably this is an issue with the JavaScript.

### PR DESCRIPTION
**Pull request by adelowo**
Fixes #4625 as per @lunny 's https://github.com/go-gitea/gitea/issues/4625#issuecomment-410907159

### CODE REVIEW

**Comment by lunny** [comment]
LGTM

**Review by appleboy — APPROVED** [review]

File: templates/repo/diff/box.tmpl [hunk 1]
```diff
@@ -8,7 +8,7 @@
 			<div class="ui right">
 				<a class="ui tiny basic toggle button" href="?style={{if .IsSplitStyle}}unified{{else}}split{{end}}">{{ if .IsSplitStyle }}{{.i18n.Tr "repo.diff.show_unified_view"}}{{else}}{{.i18n.Tr "repo.diff.show_split_view"}}{{end}}</a>
 				<a class="ui tiny basic toggle button" data-target="#diff-files">{{.i18n.Tr "repo.diff.show_diff_stats"}}</a>
-				{{if $.SignedUserID}}
+				{{if (.PageIsPullFiles) and $.SignedUserID}}
```
Comment (lafriks) [inline templates/repo/diff/box.tmpl]: Shouldn't this be: ` and .PageIsPullFiles $.SignedUserID`?

**Review by lafriks — CHANGES_REQUESTED** [review]
See my comment

File: templates/repo/diff/box.tmpl (hunk 1 above)
Comment (adelowo) [reply templates/repo/diff/box.tmpl]: Fixed @lafriks

**Review by lafriks — APPROVED** [review]

**Pull request merged**